torch/nn/_functions/rnn.py

```diff
@@ -27,8 +29,8 @@ def LSTMCell(input, hidden, w_ih, w_hh, b_ih=None, b_hh=None):
     if input.is_cuda:
         igates = F.linear(input, w_ih)
         hgates = F.linear(hidden[0], w_hh)
-        state = fusedBackend.LSTMFused.apply
-        return state(igates, hgates, hidden[1]) if b_ih is None else state(igates, hgates, hidden[1], b_ih, b_hh)
+        # Slice off the workspace arg (used only for backward)
+        return _cuda_fused_lstm_cell(igates, hgates, hidden[1], b_ih, b_hh)[:2]
```

Comment:
You sure you don't want to do this in C++? :)

Reply:
Not sure what you mean. I'll be porting this file to C++ in a next PR, but I didn't want to bundle everything in a single huge patch.